src/resspect/fit_lightcurves.py

```diff
@@ -391,6 +391,7 @@ def request_TOM_data(url: str = "https://desc-tom-2.lbl.gov", username: str = No
         dic['mjd_now'] = mjdnow
     if cheat_gentypes is not None:
         dic['cheat_gentypes'] = cheat_gentypes
+    dic['include_hostinfo'] = True
```

Comment:
@AmandaWasserman I wanted to check with you to make sure this is ok to keep here. I assume that this will just pull down more data from the TOM and that it wouldn't hurt anything, but if you can confirm that, I would appreciate it.